### PR TITLE
Enabled TTS for es-MX, eu, nb, pt-BR and mr

### LINKED ISSUE #5032: [TTS] Not all available languages are included on the app
_**Describe the issue**
I used heavily OM this past month and for some reason it seems TTS instructions in PT-BR were using PT strings (or maybe old PT-BR strings which I know I corrected long time ago). I checked data/strings/sound.txt and data/sound-strings/pt-BR.json/localize.json; both are correct._

_**System information:**
iOS last version._

Update:

**I think I discovered the actual problem. [Some languages available on the code](https://github.com/organicmaps/organicmaps/tree/master/data/sound-strings) are actually not available on the app! be, es-mx, eu, nb, pt-br, sw and probably some others (didn't check all).**

### PR DESCRIPTION
**Pull request by biodranik**
Fixes #5032

@matheusgomesms @Jean-BaptisteC can you please test these languages in TTS?

### CODE REVIEW

File: platform/languages.hpp [hunk 1]
```diff
@@ -7,13 +7,9 @@
...
 
-namespace routing
+namespace routing::turns::sound
 {
-namespace turns
-{
-namespace sound
-{
-std::array<std::pair<std::string, std::string>, 33> const kLanguageList =
+constexpr std::array<std::pair<std::string, std::string>, 37> const kLanguageList =
```
Comment (vng) [inline platform/languages.hpp:12]: и constexpr и const ?

**Review by vng — APPROVED** [review]

**Comment by matheusgomesms** [comment]
Lovely! I only have an iPhone/Mac, any way I could test it?

**Comment by biodranik** [comment]
@matheusgomesms can you build Organic Maps using XCode and run it on your device?

**Comment by matheusgomesms** [comment]
> @matheusgomesms can you build Organic Maps using XCode and run it on your device?

Yep, will see that tomorrow.

My considerations so far:

1 - Shouldn't android/res/values/strings-tts.xml be also updated?
2 - Although I understand that en/ru are the main languages of OM users, I would put all languages in alphabetical order, also to be related to Android files (they are organized like this).
3 - I would use _Português (Brasil)_ instead of _Português brasileiro_. Same for _Español (México)_, but for ES-MX I don't know if it was intentional to encapsulate all Latin America. Also _Norsk (Bokmål)_ or _Norsk Bokmål_.

**Comment by biodranik** [comment]
Thanks, fixed. Testing is required, I did not test it.

**Review by vng — APPROVED** [review]

**Pull request merged**